src/pypromice/process/L1toL2.py

```diff
@@ -217,6 +217,7 @@ def toL2(
     
 
     ds = clip_values(ds, vars_df)
+    ds = fill_gaps(ds)
```

Comment:
Are the nan values still available in the output files or are they removed later in the pipeline?

[write.py#L166 `Lcsv = Lx.to_dataframe().dropna(how="all")`](https://github.com/GEUS-Glaciology-and-Climate/pypromice/blob/develop/src/pypromice/process/write.py#L166)

[write.py#L471 `df = df.dropna(how="all")`](https://github.com/GEUS-Glaciology-and-Climate/pypromice/blob/develop/src/pypromice/process/write.py#L471)